nav2_rviz_plugins/include/nav2_rviz_plugins/flat_weighted_arrows_array.hpp

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2012, Willow Garage, Inc.
+ * Copyright (c) 2018, Bosch Software Innovations GmbH.
+ * Copyright (c) 2020, Sarthak Mittal
```

Comment:
Remove your name in the BSD license, you can only apply a license once and below is Apache 2.

Reply:
Ah, sorry about this! I'd removed it from one file and forgot to remove it from the other files.